IVOAArchitecture.tex

```diff
@@ -410,17 +410,17 @@ \section{Semantics Standards}
 places: From common designations of units to labels for physical
 quantities, from common names of reference frames and time scales to
 mutually understandable subject categories, from relationship types
-between VO resources (``this service publishes images from A and spectrafom B'') to fixed names for the messengers that produced the signals
+between VO resources (``this service publishes images from A and spectra from B'') to fixed names for the messengers that produced the signals
```

Comment:
since it's typo fixing, can you also split this line (and 423, 487 below)?
The auto wrapper I used seems to have missed them.

Reply:
Done.